libraries/UHS_host/UHS_host_INLINE.h

```diff
@@ -198,6 +198,11 @@ uint8_t UHS_USB_HOST_BASE::doSoftReset(uint8_t parent, uint8_t port, uint8_t add
  *
  */
 
+#if defined(LOAD_UHS_ENUMERATION_OPT )
+
+#include LOAD_UHS_ENUMERATION_OPT
```

Comment:
Its working for me using  the Teensy36 setup. I try never to submit code that I haven't tested out first.